test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -186,6 +186,26 @@ Describe "Get-ChildItem" -Tags "CI" {
             $app.Target | Should -Not -Be $app.FullName
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Get-ChildItem.FileSystem provider.Wildcard matching behavior is the same between -Path and -Include parameters when using escape characters
```powershell
Expected exactly '/private/tmp/8451b326-b710-488c-93eb-c99a7258d12b/a`[b]', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1: line 200
200:             $WithInclude.Name | Should -BeExactly $expectedPath
```